OntologyRetriever/annotate_abstract.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+#############################################################################################################
+#   Retrieves Pudmed Articles                                                                               #
+#                                                                                                           #
+#   Entrez REST API:                                                                                        #
+#   https://eutils.ncbi.nlm.nih.gov/entrez/eutils/                                                          #
+#                                                                                                           #
+#   Example:                                                                                                #
+#   https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?db=pubmed&term=science[journal]              #
+#   +AND+breast+cancer+AND+2008[pdat]                                                                       #
+#                                                                                                           #
+#############################################################################################################
+
+
+# In[2]:
+
+
+import ontology_retriever as onRe
+
+print("Retrieving all concepts in the following list of ontologies: ", onRe.list_of_bioportal_ontologies)
+concepts = onRe.retrieve_annotations("dde26f65-ec2b-49b4-b3e7-090e5da33350", max_page_limit=10)  
+
+
+# In[3]:
+
+
+import requests
+import xmltodict
+
+search_keyword="bipolar"
```

Comment:
Could we replace this with "bipolar disorder" as per [this](https://github.com/8hk/AEK/issues/15#issuecomment-739152601)?